src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/FileResultExecutorBase.cs

```diff
@@ -255,7 +258,7 @@ private static void SetAcceptRangeHeader(HttpResponse response)
                 var state = matchNotFoundState;
                 foreach (var entityTag in etagHeader)
                 {
-                    if (entityTag.Equals(EntityTagHeaderValue.Any) || entityTag.Compare(etag, useStrongComparison: true))
+                    if (entityTag.Equals(EntityTagHeaderValue.Any) || entityTag.Compare(etag, useStrongComparison: useStrongComparison))
```

Comment:
nit: you can drop the named parameter here, it's already clear what you're doing.